jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -35,12 +37,18 @@ function configureEnv() {
 }
 
 function configure() {
+    configure_mem_ratio
     configure_router_state
     configure_router_location
     configure_controller_access
     configure_router_tls
 }
 
+function configure_mem_ratio() {
+    export JAVA_MAX_MEM_RATIO=80
+    export JAVA_INITIAL_MEM_RATIO=25
```

Comment:
didn't see the changes here, this way you are blocking user to customize this env, 80 and 25 should be the default value, not hardcoded.
as I said before use something line this:

```
 export JAVA_MAX_MEM_RATIO=${JAVA_MAX_MEM_RATIO:-80}
```
and in the tests, create tests using different values than the default, test it passing nothing to make sure the default value will be applied...

Reply:
done